Makefile

```diff
@@ -0,0 +1,19 @@
+# Build an executable for running go server after database migrations pass
+# 
+SHELL := /bin/bash
+
```

Comment:
Made it because of problem with mysql server and running migrations, in mysql container database running later that go container trying run migrations. In general we don't need all of this unless its going to prod )